Update changelog. (#57)
twilio/twilio-voice.js: CHANGELOG.md

The SDK now fully supports Call reconnection. Previously, the SDK only truly supported media reconnection -- if the
media connection was lost but the signaling websocket recovered (generally within 10-15 seconds), reconnecting

> the Call was possible. However, if the websocket was lost, the Call was lost. Now, the SDK is able to reconnect

Reviewer: ```suggestion
the Call was possible. However, if the websocket was lost due to a network handover event, the Call was lost. Now, the SDK is able to attempt to reconnect
```

> a Call even if the websocket is lost. This means that a Call can potentially be recovered up to 30 seconds after a network handover event.

Reviewer: ```suggestion
a Call for up to 30 seconds after a websocket is lost. 
```
